README.md

```diff
@@ -6,7 +6,7 @@
 
 This repo contains the following clients:
   * [NuGet CLI](https://docs.nuget.org/ndocs/tools/nuget.exe-cli-reference)
-  * [NuGet Package Manager for Visual Studio 2017](https://docs.nuget.org/ndocs/tools/package-manager-ui)
+  * [NuGet Package Manager for Visual Studio](https://docs.nuget.org/ndocs/tools/package-manager-ui)
```

Comment:
https://docs.microsoft.com/en-us/nuget/tools/package-manager-ui?

Reply:
The links are fwed but I can double check them.

Reply:
All the links go where they are supposed to.

Reply:
They are forwarded, but there's a higher risk of the docs.nuget.org link breaking than the current docs URL, unless you think nuget's docs are going to move back to nuget.org at some point in the future.

Reply:
Good point, I missed that it's docs.**nuget**.

Will fix that.